addons/hr_homeworking_calendar_google/data/ir_cron_data.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <data noupdate="1">
+        <record id="sunday_sync_cron" model="ir.cron">
+            <field name="name">HR Homeworking Calendar Google: sync working locations</field>
+            <field name="model_id" ref="hr_homeworking_calendar_google.model_res_users"/>
+            <field name="state">code</field>
+            <field name="code">model.cron_sync_working_locations()</field>
+            <field name="user_id" ref="base.user_root"/>
+            <field name="interval_number">7</field>
```

Comment:
Maybe 7 days are too much. Could it cause issue if a lot of user are created or if homeworking locations are updated and the odoo code is still using the old one ?

Reply:
It depends on the value we have on start and end. too. I am not sure. we can think about it and decide at review.